src/rental/rental.service.ts

```diff
@@ -108,11 +121,27 @@ export class RentalService {
     return result;
   }
 
-  update(id: number, updateRentalDto: UpdateRentalDto) {
-    return `This action updates a #${id} rental`;
+  async updateRental(
+    id: string,
+    updateRentalDto: UpdateRentalDto,
+  ): Promise<Rental> {
+    const updated = await this.rentalModel.findByIdAndUpdate(
+      id,
+      updateRentalDto,
+      { new: true },
+    );
+
+    if (!updated) {
+      throw new NotFoundException('Rental not found');
```

Comment:
chỗ này sửa lại là with id not found nhen

Reply:
oki ông